test/utils.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "baldr/rapidjson_utils.h"
+#include "worker.h"
+
+#include <boost/property_tree/ptree.hpp>
+#include <sstream>
+
+namespace test {
+
+inline boost::property_tree::ptree json_to_pt(const std::string& json) {
+  std::stringstream ss;
+  ss << json;
+  boost::property_tree::ptree pt;
+  rapidjson::read_json(ss, pt);
+  return pt;
+}
+
+inline rapidjson::Document to_document(const std::string& request) {
+  rapidjson::Document d;
+  auto& allocator = d.GetAllocator();
+  d.Parse(request.c_str());
+  if (d.HasParseError())
+    throw valhalla::valhalla_exception_t{100};
```

Comment:
this seems a bit strange, can you mention why you would want to use this exception rather than just a generic one? these exceptions are to be used in request processing. i suspect this here is only used for test cases?

Reply:
I've just moved this function from several cpps into header file in order to avoid copying

Reply:
generally, this function it's not used anywhere. thinks, it's better to remove it at all